week-4/C-more-complex-objects/exercise-3.js

```diff
@@ -0,0 +1,54 @@
+/*
+    Given the same "house" object again
+
+    Write the code for the functions as per the description above them
+*/
+
+var house = {
+    address: '1 Kinning Park',
+    price: 180000,
+    currentOwner: {
+      firstName: 'Margaret',
+      lastName: 'Conway',
+      email: "margaret@fake-emails.com"
+    }
+};
+
+var house = {

```

Comment:
You've defined `house` twice. In the code below you use `house1` and `house2`.

Reply:
good spot!!